### ISSUE TITLE
The URL on which the test must run should be configurable

### ISSUE THREAD

**Issue by as33ms** [apinf/apinf-testing#13]
Since the tests will be run on different apinf installations, it should be possible configure the URL either in the settings or as an environment variable `e.g. INSTALLATION_URL=http://nightly.apinf.io` or in Meteor settings: `installation_url`.

**Comment by as33ms** [comment]
At this moment, the URL is hardcoded to `nightly.apinf.io` at https://github.com/apinf/apinf-testing/blob/master/test/common.js#L5

**Comment by mauriciovieira** [comment]
I don't think it is a good idea to have a separate repository for automated tests. The tests should be in the developer workflow, and feedback should be fast. The developers should always know if the latest change broke the tests, and in backend javascript projects the convention is just run `npm test` (or `yarn test`, since we recently [switched to yarn](https://github.com/apinf/platform/issues/2476)).

The tests are being rewritten from scratch and this repository is not being used.

**Comment by mauriciovieira** [comment]
After sending the last comment, I take back a bit of what I said. I *do* think that some automated tests should be in a separate repository. But only non-functional tests make sense to be run here like:

* stress tests
* security tests

What do you think?
